k8s/README.md

```diff
@@ -56,3 +59,63 @@ and add two lines in the `metadata.annotations`
 
 - `service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"`
 - `service.beta.kubernetes.io/do-loadbalancer-hostname: "cloudbeaverBaseDomain"`
+
+### Clouds volumes configuration
+
+To store Team Edition data in the cloud, you need to configure cloud volumes. For example, you can store connection configurations and user information in AWS EFS.
+
+Once this is set up, you can deploy Team Edition by following [this guide](#how-to-run-services).
+
+#### AWS
+
+##### Prerequisites
+
+- **AWS CLI** installed and configured
+- **eksctl** installed
+- **Helm** installed
+- **Terraform** installed
+- Access to an existing **EKS cluster**
+
+
+##### Step 1: Associate IAM OIDC Provider
+
+Associate the IAM OIDC provider with your EKS cluster to enable IAM roles for service accounts.
+
+```
+eksctl utils associate-iam-oidc-provider \
+  --region=<your-region> \
+  --cluster=<your-cluster-name> \
+  --approve
+```
+
+##### Step 2: Install AWS EFS and EBS CSI Drivers
+
+Install the AWS EFS and EBS CSI drivers using Helm.
+
+```
+helm repo add aws-efs-csi-driver https://kubernetes-sigs.github.io/aws-efs-csi-driver/
+helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver/
+helm repo update
+
+helm install aws-efs-csi-driver aws-efs-csi-driver/aws-efs-csi-driver --namespace kube-system
+helm install aws-ebs-csi-driver aws-ebs-csi-driver/aws-ebs-csi-driver --namespace kube-system
+```
+
+##### Step 3: Configure EFS via Terraform
+
+1. Navigate to Directory `team-edition-deploy/AWS/aws-eks`
+2. Open the `main.tf` file in a text editor
+3. Update the following variables with your AWS region and EKS cluster name
+```
+variable "region" {
+  description = "Region for AWS EFS"
+  default     = ""<your-region>"
+}
+variable "cluster_name" {
+  description = "EKS cluster name"
+  default     = "<your-cluster-name>"
+}
+```
+4. Run `terraform init` and `terraform apply`
```

Comment:
I got 
│ Error: creating EFS File System: operation error EFS: CreateFileSystem, https response error StatusCode: 403, RequestID: de6480bf-7b82-4e70-a316-170f4cefb3f3, api error AccessDeniedException: User: arn:aws:iam::905418347357:user/dbauser is not authorized to perform: elasticfilesystem:CreateFileSystem on the specified resource

and had to add 
AmazonElasticFileSystemFullAccess